packages/bundle/src/FullReactWebChat.js

```diff
@@ -31,10 +31,18 @@ class FullReactWebChat extends React.Component {
   }
 
   render() {
-    const { adaptiveCardHostConfig, attachmentMiddleware, styleOptions, styleSet, ...otherProps } = this.props;
+    const {
+      adaptiveCardHostConfig,
+      attachmentMiddleware,
+      styleOptions,
+      styleSet,
+      renderMarkdown,
+      ...otherProps
+    } = this.props;
 
     const memoizedStyleSet = this.memoizeStyleSet(styleSet, styleOptions);
-    const memoizedRenderMarkdown = this.memoizeRenderMarkdown(renderMarkdown, memoizedStyleSet);
+    const memoizedRenderMarkdown =
+      renderMarkdown || this.memoizeRenderMarkdown(defaultRenderMarkdown, memoizedStyleSet.options.markdown);
```

Comment:
I think `memoizedStyleSet.options.markdown` could be reduced to `styleOptions ? styleOptions.markdown : {}`. I am not 100% sure but sounds like it might work.

Less dot is better because member accessors in JavaScript is super expensive due to their take on prototypical OO model.